Description: Minor build improvements (bump API version, fix links, better bundle names)
Author: mcculls@gmail.com
Bug-Google: http://code.google.com/p/google-guice/issues/detail?id=720
Last-Update: 2012-07-29

diff --git a/build.xml b/build.xml
index fa2d666..04667e0 100644
--- a/build.xml
+++ b/build.xml
@@ -248,6 +248,9 @@
         <include name="NOTICE"/>
       </zipfileset>]]></replacetoken>
     </replace>
+    <replace file="build/no_aop/common.xml" value='Bundle-Name" value="$${ant.project.name} (no_aop)'>
+      <replacetoken><![CDATA[Bundle-Name" value="${ant.project.name}]]></replacetoken>
+    </replace>
   </target>
 
   <target name="clean.all"
diff --git a/common.xml b/common.xml
index 576ead3..d6dbd19 100644
--- a/common.xml
+++ b/common.xml
@@ -7,7 +7,7 @@
   <!-- can be overridden at the command line with -Dversion=
        or in IDEA, in the ant properties dialog -->
   <property name="version" value="snapshot"/>
-  <property name="api.version" value="1.3"/>
+  <property name="api.version" value="1.4"/>
 
   <target name="compile" description="Compile Java source.">
     <mkdir dir="${build.dir}/classes"/>
diff --git a/core/pom.xml b/core/pom.xml
index 4289914..8dc1fda 100644
--- a/core/pom.xml
+++ b/core/pom.xml
@@ -109,6 +109,7 @@
         <artifactId>maven-bundle-plugin</artifactId>
         <configuration>
           <instructions>
+            <Bundle-Name>${project.artifactId}$(if;$(classes;NAMED;*.MethodAspect);; (no_aop))</Bundle-Name>
             <DynamicImport-Package>org.slf4j</DynamicImport-Package>
             <Eclipse-ExtensibleAPI>true</Eclipse-ExtensibleAPI>
           </instructions>
diff --git a/pom.xml b/pom.xml
index 2c5dee0..2cf1023 100644
--- a/pom.xml
+++ b/pom.xml
@@ -61,9 +61,9 @@ See the Apache License Version 2.0 for the specific language governing permissio
   </mailingLists>
 
   <scm>
-    <connection>scm:svn:http://google-guice.googlecode.com/svn/trunk</connection>
-    <developerConnection>scm:svn:https://google-guice.googlecode.com/svn/trunk</developerConnection>
-    <url>http://code.google.com/p/google-guice/source/browse</url>
+    <connection>scm:git:https://code.google.com/p/google-guice/</connection>
+    <developerConnection>scm:git:https://code.google.com/p/google-guice/</developerConnection>
+    <url>https://code.google.com/p/google-guice/</url>
   </scm>
 
   <issueManagement>
@@ -73,7 +73,7 @@ See the Apache License Version 2.0 for the specific language governing permissio
 
   <ciManagement>
     <system>Hudson</system>
-    <url>https://grid.sonatype.org/ci/job/Google-Guice/</url>
+    <url>https://builds.sonatype.org/job/google-guice/</url>
   </ciManagement>
 
   <licenses>
@@ -97,7 +97,7 @@ See the Apache License Version 2.0 for the specific language governing permissio
     <!--
      | The spec version of the public Guice API
     -->
-    <guice.api.version>1.3</guice.api.version>
+    <guice.api.version>1.4</guice.api.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!--
      | Use "-Dguice.with.jarjar=false" to build without jarjar
